tests/unit/literaturesuggest/test_literaturesuggest_tasks.py

```diff
@@ -50,9 +53,39 @@ def __init__(self, email):
         self.email = email
 
 
+@pytest.fixture
+def mock_perform_es_search_onerecord():
+    return result.Response(
+        {
+            "hits": {
+                "hits": [
+                    {
+                        "_index": "records-hep",
+                        "_type": "hep",
+                        "_id": "42",
+                        "_score": 11.123,
+
+                        "_source": {
+                            "short_titles": [
+                                {
+                                    "title": "ShortTitle"
+                                }
+                            ]
+                        }
+                    }
+                ],
+                "max_score": 11.123,
+                "total": 1
+            },
+            "timed_out": False,
+            "took": 123
+        }
+    )
+
+
 @mock.patch('inspirehep.modules.literaturesuggest.tasks.User')
 @mock.patch('inspirehep.modules.literaturesuggest.tasks.UserIdentity')
-def test_formdata_to_model_populates_schema(u, ui):
+def test_formdata_to_model_populates_schema(u, ui, app):
```

Comment:
There's a better solution for this: readd `autouse=True` to the `app` fixture, and remove all current explicit uses of it by using `current_app` instead. This requires a little more work, but will avoid this problem from now on.